client/landing/jetpack-cloud/sections/settings/main.jsx

```diff
@@ -27,55 +29,96 @@ import './style.scss';
 import connectedIcon from './images/server-connected.svg';
 import disconnectedIcon from './images/server-disconnected.svg';
 
-const connectedProps = ( translate ) => ( {
+const connectedProps = ( translate, connectedMessage ) => ( {
 	iconPath: connectedIcon,
-	className: 'settings_connected',
+	className: 'settings__connected',
 	title: translate( 'Server status: Connected' ),
-	content: translate( 'One-click restores are enabled.' ),
+	content: connectedMessage,
 } );
 
-const disconnectedProps = ( translate ) => ( {
+const disconnectedProps = ( translate, disconnectedMessage ) => ( {
 	iconPath: disconnectedIcon,
-	className: 'settings_disconnected',
+	className: 'settings__disconnected',
 	title: translate( 'Server status: Not connected' ),
-	content: translate(
-		'Enter your server credentials to enable one-click restores for Backups. {{a}}Need help? Find your server credentials{{/a}}',
-		{
-			components: {
-				a: (
-					<ExternalLink
-						className="settings__link-external"
-						icon
-						href="https://jetpack.com/support/adding-credentials-to-jetpack/"
-					/>
-				),
-			},
-		}
-	),
+	content: translate( '%s {{a}}Need help? Find your server credentials{{/a}}', {
```

Comment:
This is not great for translations. Can we make the whole message right there instead?